src/lib/gup.js

```diff
@@ -0,0 +1,34 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var identity = require('./identity');
+
+function wrap(d) {return [d];}
+
+module.exports = {
```

Comment:
I like this. Thanks @monfera :tada:

Reply:
Would you mind reusing this in `parcoords`.

Reply:
Added note to https://github.com/plotly/plotly.js/issues/2058